src/services/BackgroundSchedulerService/BackgroundSchedulerService.ts

```diff
@@ -14,10 +16,14 @@ const registerPeriodicTask = (task: PeriodicTask) => {
       stopOnTerminate: false,
     },
     async taskId => {
-      await task();
-      BackgroundFetch.finish(taskId);
+      if (taskId === BACKGROUND_TASK_ID) {
+        await task();
+        BackgroundFetch.scheduleTask({taskId: BACKGROUND_TASK_ID, delay: 0, periodic: true});
```

Comment:
I don't think we need to call `scheduleTask` again. I removed it and tested. It works. 

The way I test it is: 
- Setup a dump ngrok server. 
- In the code where I want to log, do `fetch(`https://4a0b902fcbc4.ngrok.io/registerPeriodicTask/start`).catch(() => {});`.
- Below is the log for roughly every 15 mins. 

```
GET /registerPeriodicTask/taskId/app.covidshield.exposure-notification 502 Bad Gateway
GET /registerPeriodicTask/taskId/react-native-background-fetch         502 Bad Gateway
GET /registerPeriodicTask/taskId/react-native-background-fetch         502 Bad Gateway
GET /registerPeriodicTask/taskId/app.covidshield.exposure-notification 502 Bad Gateway
GET /registerPeriodicTask/taskId/react-native-background-fetch         502 Bad Gateway
GET /registerPeriodicTask/taskId/app.covidshield.exposure-notification 502 Bad Gateway
GET /registerPeriodicTask/start                                        502 Bad Gateway
GET /registerPeriodicTask/start                                        502 Bad Gateway
```